packages/jitm/src/Manager.php

```diff
@@ -177,17 +191,21 @@ function jitm_enqueue_files() {
 
 		wp_enqueue_script(
 			'jetpack-jitm-new',
-			Jetpack::get_file_url_for_environment( '_inc/build/jetpack-jitm.min.js', '_inc/jetpack-jitm.js' ),
+			\Jetpack::get_file_url_for_environment( '_inc/build/jetpack-jitm.min.js', '_inc/jetpack-jitm.js' ),
 			array( 'jquery' ),
 			JETPACK__VERSION,
```

Comment:
JETPACK_VERSION would not be present if the package is not running alongside with Jetpack